ToDoList/ToDoList/Modules/TaskManager/Core/Presenter/TaskManagerPresenter.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  TaskManagerPresenter.swift
+//  ToDoList
+//
+//  Created by Никита Данилович on 04.05.2023.
+//
+import UIKit
+
+final class TaskManagerPresenter: TaskManagerPresenterInterface {
+
+    private var view: TaskManagerViewControllerInterface?
+    var storage: StorageInterface
```

Comment:
```suggestion
    private let storage: StorageInterface
```
Let's make it constant as it won't change during the life cycle of presenter

Reply:
it's declared in the TaskManagerPresenterInterface, we are not able to do it

Reply:
But do we need it in TaskManagerPresenterInterface